ffv1.md

```diff
@@ -703,6 +703,9 @@ Default values at the decoder initialization phase:
 
 ## Parameters
 
+A `Parameters` element provides information about the decoding configuration of the frame (version 0 and 1) or the whole stream (other versions).
```

Comment:
backtick quote Frame. Also throughout the document it seems we've been preferring to say 'bitstream' (or 'bytestream') rather than simply 'stream'.